trieste/acquisition/function.py

```diff
@@ -509,6 +512,161 @@ def prepare_acquisition_function(
         return lambda at: expected_improvement(objective_model, eta)(at) * constraint_fn(at)
 
 
+class ExpectedHypervolumeImprovement(SingleModelAcquisitionBuilder):
+    """
+    Builder for the :func:`expected_hv_improvement` acquisition function.
```

Comment:
this is a builder for the expected hypervolume improvement acquisition function as opposed to `expected_hv_improvement`, which isn't an acquisition function. Similarly below, `prepare_acquisition_function` returns the acquisition function rather than `The expected_hv_improvement function`